index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/index.css" />
+    <title>Microverse Portfolio</title>
+  </head>
+  <body>
+    <section class="nav-section relative">
+      <nav class="w-full">
+        <div class="flex align-items-center h-4">
+          <div class="nav-items text-center">
+            <a href="#"><span class="fw-600">Portfolio</span></a>
+            <a href="#"><span class="fw-600">About</span></a>
+            <a href="#"><span class="fw-600">Contact</span></a>
+          </div>
+          <div class="nav-container">
+            <input class="checkbox" type="checkbox" />
+            <div class="menu-button">
+              <span class=""></span>
+              <span class=""></span>
+              <span class=""></span>
+            </div>
+          </div>
+        </div>
+      </nav>
```

Comment:
To make your page more semantic, I suggest your navigation bar should be wrapped in a `header` tag rather than a `section` tag😉 Kindly take away the `section` tag wrapping the `nav` and replace it with  a `header` tag👍